tools/ratbagctl.in

```diff
@@ -719,7 +719,7 @@ parser_def = [
                     func: func_led_get,
                 },
                 {
-                    of_type: set,
+                    of_type: switch,
```

Comment:
Well, this is not how the command are supposed to work (the fact that it complains is not correct):
- `ratbagctl led 0 set mode on color 00ff00 event6`
- `ratbagctl led 0 set mode cycle rate 8000 event6`

It works for me with a commit that adds calls to the commit API (will send the PR soon).